dashboard/test/serializers/api/v1/pd/enrollment_flat_attendance_serializer_test.rb

```diff
@@ -75,4 +75,40 @@ class Api::V1::Pd::EnrollmentFlatAttendanceSerializerTest < ::ActionController::
     }
     assert_equal expected, serialized
   end
+
+  test 'deleted user that attended' do
```

Comment:
nit: indicate in the test case name what it expects, e.g. `deleted user that attended is included`